### PR TITLE
Adding Layout updates across the Demo site

### PR DESCRIPTION
**Pull request by fellyph**
## Description

This PR contains the suggested layout changes from #13. It includes a new shared header with a small logo on the left side with a link to go back to the home. This pull request also includes a new structure on the home page with one single section unifying API Demos and Scenarios. 

The changes made on the header component affect all other pages because the main container was moved to the header file. In this case, putting an extra tag on each view wasn't necessary. That is why the other views contain changes.

## Relevant Technical Choices
- Ensure the layout is consistent across the home page and all the demos with Tailwind CSS
- Add more space between the visual elements
- Creating a button component to reduce the duplication of code over the home page
- Removing the duplication of the header and footer by the iframe content with a variation of the header and the footer
- Adding a base structure to the single demo page to have a consistent structure

## Additional information

- The Layout does not use the color scheme from the Privacy Sandbox. We can apply a theme on the subsequent pull request with primary and secondary colors
- The changes applied to the button only included a 1x1 aspect ratio. This also can be improved using the new color scheme

## Screenshot

**Homepage before**
home page before

**Homepage**
demo site - homepage

** Analytics demo page before**
demo page before

**Analytics demo page**
demo site - analytics

**E-commerce demo page**
demo site - ecommerce

**Payment demo page**
demo site - payment

**SSO demo page**
demo site - sso

**Mobile Version**
mobile version home page

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,8 @@
 protocol=http # http or https
 domain-a=localhost # domain-aaa.com
-domain-a-background=bg-green-100
+domain-a-background=bg-sky-50
```
Comment (gagan0123) [inline .env.example:3]: Reason I selected green is to provide a clear context that green means allowed and anything appearing green corresponds to what should be allowed in the page

Comment (fellyph) [reply .env.example:3]: @gagan0123 the `sky-50` is close to the blue background color on Privacy Sandbox website

Comment (fellyph) [reply .env.example]: Perfect I have reverted it

**Comment by pradeep910** [comment]
@fellyph Can you pls make changes like removing the title from homepage, etc. and resolve conflicts so we can merge it?

**Pull request closed without merging**